packages/backend-core/src/middleware/joi-validator.ts

```diff
@@ -6,7 +6,7 @@ function validate(
   property: string,
   opts?: { errorPrefix?: string; allowUnknown?: boolean }
 ) {
-  const errorPrefix = opts?.errorPrefix || `Invalid ${property}`
+  const errorPrefix = opts?.errorPrefix ?? `Invalid ${property}`
```

Comment:
What's the difference between `||` and `??` here?

Reply:
`||` is a [logical or](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR), so it "converts" the values to boolean. `??` is a [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing), checks only for nulls or undefineds.

Main difference:
'' || 'default' -> 'default'
'' ?? 'default' -> ''